### ISSUE TITLE
Unresolved dependencies kundera-core in kundera-spark 3.x

### ISSUE THREAD

**Issue by anand-singh** [Impetus/kundera#820]
Error:Unresolved dependencies:<ul><li> com.impetus.kundera.core#kundera-core;3.0-SNAPSHOT: not found</li></ul>

**Comment by devender-yadav** [comment]
@anand-singh 

Don't use `3.0-SNAPSHOT`.  `SNAPSHOT` is **development version**. Try stable version. Latest is 3.2

-Dev

**Comment by anand-singh** [comment]
@devender-yadav 

I am using 3.2 but it picks the old one internally

**Comment by devender-yadav** [comment]
@anand-singh 

It should pick `com.impetus.kundera.core#kundera-core;3.2`. 
- Which module are you using? 
- Can you share the exact dependency tried?
- Are you adding Kundera-core dependency externally?

-Dev

**Comment by anand-singh** [comment]
@devender-yadav 

I am trying to add below deps:

```
"com.impetus.kundera.client" % "kundera-spark" % "3.2"
"com.impetus.kundera.client" % "kundera-mongo" % "3.2"
```

Not using kundera-core directly.

Thanks,
Anand

**Comment by devender-yadav** [comment]
@anand-singh 

This is a problem related to Kundera-Spark module. We will fix it ASAP. 

-Dev

**Comment by anand-singh** [comment]
@devender-yadav  thanks 

Changing the issue descriptions

**Comment by devender-yadav** [comment]
@anand-singh 

Fix has been added for this issue & it will be available from the next release(3.3). Meanwhile, you can build from trunk.

-Dev

**Comment by chhavigangwal** [comment]
Fixed with Kundera-3.3
